crates/turborepo-lib/src/run/scope/target_selector.rs

```diff
@@ -195,8 +195,12 @@ pub fn is_selector_by_location(
             .iter()
             .any(|prefix| raw_selector.starts_with(prefix))
     {
+        let cleaned_selector = path_clean::clean(std::path::Path::new(raw_selector))
```

Comment:
We do this to match the Go behavior where `turbopath.MakeRelativeSystemPath` called `filepath.Clean` under the hood which would remove any leading `./`.

Reply:
Maybe we should clean all AnchoredSystemPathBuf and RelativeUnixPaths such that there are as few ways to represent the same path as possible.